app/middlewares/userValidator.js

```diff
@@ -0,0 +1,9 @@
+const userValidator = require('../services/userValidator');
+
+module.exports = (request, response, next) => {
+  if (userValidator(request.body)) {
+    next();
+  } else {
+    response.status(403).send('The information is invalid');
```

Comment:
I'll add some information here about the errors

Reply:
This is related to one of the "known issues" I added. My first validator returned all the errors but it wasn't specified in the story. So, should I send all the errors or maintain the actual functionality and send the description of the first produced error?

Reply:
For someone who uses this API will be more clear if you send all the errors. You can choose the format for sending them